src/state.js

```diff
@@ -0,0 +1,73 @@
+import Project from './project'
+import Task from './task'
+
+let db = firebase.firestore()
+const docRef = db.collection('projects').doc('Naqhid')
+
+export default function saveState() {
+  // localStorage.setItem('projects', JSON.stringify(window.projects));
```

Comment:
Comments: It’s better to delete commented pieces of code. If you need to review a previous version of your code, use git history.
 
 The previous reviewer meant you should remove this commented code.